app/models/profile_field_group.rb

```diff
@@ -0,0 +1,5 @@
+class ProfileFieldGroup < ApplicationRecord
+  has_many :profile_fields, dependent: :nullify
```

Comment:
I went for `nullify` here because I wasn't 100% sure if all fields always need to belong to a group.